velox/functions/sparksql/aggregates/tests/SumAggregationTest.cpp

```diff
@@ -28,6 +32,82 @@ class SumAggregationTest : public SumTestBase {
     SumTestBase::SetUp();
     registerAggregateFunctions("spark_");
   }
+
+ protected:
+  // check global partial agg overflow, and final agg output null
```

Comment:
Please capitalize the first letter and add period (.) at the end for comments. Thanks.